SI/Sem.03/Pract.03/Task05.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+	int n;
+	cout << "enter initial number" << endl;
+	cin >> n;
+	int cur = -1;
+	int guesses = 0;
+	while (true) {
```

Comment:
avoid while (true).
- what is the condition?
- will we always have at least 1 try?
- do we need to check for other conditions?